apps/graphql/src/apps/location/types/enums/LocationType.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import { GraphQLEnumType } from 'graphql';
+
+export default new GraphQLEnumType({
+  name: 'LocationType',
+  values: {
+    station: {
```

Comment:
Values should be UPPER_CASED

Reply:
So this it is the convention for Enums ?

Reply:
seems like it yeah, all enums in `incubator/universe` follow this convention, and all enums we have in margarita so far.